week-1/Homework/extra/2-piping.js

```diff
@@ -17,7 +17,7 @@
 */
 
 function add(num1 ,num2) {
-const sum=(num1+num2);
+const sum=(num1*10+num2*10)/10;
```

Comment:
Hi Tas,
How come you are multiplying all the numbers by `10` and then dividing them by `10` afterwards ?